### PR TITLE
reword and remove checkbox in github template

### PR DESCRIPTION
**Pull request by Jason0214**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `Closes #xxx[,#xxx]`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**
reword and remove checkbox in github template, checkbox is used to report progress, use it to describe PR information would make the PR look like not complete,

as you can see
image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -4,16 +4,19 @@ PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATI
...
-[ ] Performance issue
-[ ] Regression (a behavior that used to work and stopped working in a new release)
-[ ] Documentation issue or request
-[ ] Other... Please describe:
+<!-- Please search GitHub for a similar issue or PR before submitting -->
+
+# Shortest description
+
+<!-- What you are submitting ?
+e.g.
```
Comment (Armour) [inline .github/ISSUE_TEMPLATE.md]: E.g. ? (capitalize)

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -4,16 +4,19 @@ PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATI
...
+    bug report
+    feature request
+    performance issue
+    regression (a behavior that used to work and stopped working in a new release)
+    documentation issue or request
+    or others... Please describe
```
Comment (Armour) [inline .github/ISSUE_TEMPLATE.md]: or others... (please describe)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -4,30 +4,18 @@ https://github.com/Armour/jarvis/blob/master/.github/CONTRIBUTING.md#submitting-
...
-- [ ] Build-related changes
-- [ ] Other, please describe:
+- When resolving a specific issue, make sure it's referenced in the PR's title (e.g. `Closes #xxx[,#xxx]`, where "xxx" is the issue number)
 
-**Does this PR introduce a breaking change?** (check one)
+- If adding a **new feature**, the PR's description includes: A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)
 
-- [ ] Yes
-- [ ] No
+- If this PR introduce a breaking change, please describe the impact and migration path for existing applications
```
Comment (Armour) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```
**breaking change**
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -4,30 +4,18 @@ https://github.com/Armour/jarvis/blob/master/.github/CONTRIBUTING.md#submitting-
...
+**What kind of change does this PR introduce?**
 
-**The PR fulfills these requirements:**
-
-- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `Closes #xxx[,#xxx]`, where "xxx" is the issue number)
-
-If adding a **new feature**, the PR's description includes:
-
-- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)
+<!-- e.g. bugfix, feature, code style update, refactor, build-related changes,
```
Comment (Armour) [inline .github/PULL_REQUEST_TEMPLATE.md]: try to match ISSUE_TEMPLATE.md format, put those types in separate lines.

**Review by Armour — CHANGES_REQUESTED** [review]
一些小建议hhh

**Comment by Jason0214** [comment]
fixed @Armour

**Review by Armour — APPROVED** [review]

**Pull request merged**